### ISSUE TITLE
[Bug]: Strange behavior when DB tab is pinned

### ISSUE THREAD

**Issue by aparente** [RafaelGB/obsidian-db-folder#888]
### What happened?

I have recurring issues with different db functions when a tab is pinned. 

I like keeping the database tab pinned because it lets me open other files while keeping the db open. 

However, I've run into a number of weird issues that don't happen _all_ the time but do happen frequently. These are resolved by closing/re-opening the database, but its quite frustrating!

- search stops working, I can see the file name but searching for it returns no results 
- table doesn't update, either when files get re-named or when a cell is filled in  
- relations don't work as expected, though this might just be me being dumb because it doesn't just happen when a tab is pinned (though I have had a case where closing/reopening the db updates the relation). they seem to work sometimes but not all the time. 

I know I've filed a few bug reports today, but just wanna say this is a super useful tool and I'm already getting a lot of utility out of it. Thank you!

### What platform were you using?

Desktop

### Version of the plugin

3.3.3

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by shuibingchenlab** [comment]
I also have the same issue - when pinned the page - nth now appear in the table until i open it agian in a new tab

**Comment by bonecast** [comment]
yeah same here i usually open settings and close real fast for the reset.
